TCAT/Controllers/AllStopsTableViewController.swift

```diff
@@ -164,5 +173,16 @@ class AllStopsTableViewController: UITableViewController {
     @objc func backAction() {
         navigationController?.popViewController(animated: true)
     }
-
+    
+    // MARK: DZNEmptyDataSet
+    
+    func image(forEmptyDataSet scrollView: UIScrollView!) -> UIImage! {
+        return #imageLiteral(resourceName: "road")
+    }
+    
+    func description(forEmptyDataSet scrollView: UIScrollView!) -> NSAttributedString! {
+        let title = "Couldn't get stops 😟 "
```

Comment:
Love this (we use them elsewhere too 🙂), just get ride of that one space at the end!